config-example.properties

```diff
@@ -42,6 +42,9 @@ graph.dataaccess=RAM_STORE
 # graph but less for every request.
 # prepare.chWeighting=fastest
 #
+# You can also setup multiple weightings, by providing a coma separated list like:
+# prepare.chWeighting=fastest, shortest
```

Comment:
> Do you think adding logic to a setter is bad?

Yeah. That is the only thing I don't like about it. But when reading the config file we should trim and then set with the array which is then used 'as it'. Maybe even do not do toLowerCase but throw exception if upper case?